<launch>
  <node name = "screw_subscriber" pkg = "GPIO_output" type = "screw_sub.py" output = "screen">
    <param name = "pwm_screw" value = "100"/>
    <param name = "duration_screw" value = "3" />
  </node>

  <node name = "gripper_subscriber" pkg = "GPIO_output" type = "gripper.py" output = "screen">
    <param name = "duration_gripper" value = "3" />
  </node>

  <node name = "sonar_publisher" pkg = "GPIO_output" type = "sonar_pub.py" output = "screen"/>

</launch>
